src/textual/_animator.py

```diff
@@ -5,19 +5,22 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from functools import partial
-from typing import TYPE_CHECKING, Any, Callable, TypeVar
+
+from typing import Callable
+from typing_extensions import (
```

Comment:
Shouldn't we import as much as possible from `typing`? E.g., `TYPE_CHECKING` and `Any` are definitely available in `typing`, right?

Reply:
I see this has been fixed meanwhile.